post/database/factories/PostFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| Here you may define all of your model factories. Model factories give
+| you a convenient way to create models for testing and seeding your
+| database. Just tell the factory how a default model should look.
+|
+*/
+
+$factory->define(App\Post::class, function (Faker\Generator $faker) {
+    return [
+        'title'     => $faker->text(20),
+        'body'    => $faker->text(200)
```

Comment:
If you want to add indentation between array index and value at least try to align them. Or use just one space delimiter